OneCommunityGlobal/HighestGoodNetworkApp
# Akshay Changes Alan Weekly Summaries

# Description
This PR is a replacement for Alan's PR #593 and includes all the changes from that PR, as well as a few minor updates, such as removing unused comments.

## Main changes explained:
- Implement the functionality from Alan's PR #593
- Remove unused comments from various files in the codebase

## How to test:
- Check out the current branch
- Run `npm install` to ensure all dependencies are installed
- Start the application and verify that it runs smoothly and that the new functionality from PR #593 is working correctly

## Screenshots or videos of changes (if applicable):
None, as the changes are only related to comments and do not affect UI or functionality.

Note:
This PR includes the changes from Alan's PR #593 and additional changes related to removing unused comments. The code has been tested locally, and no issues have been identified.
